packages/react/src/date-picker/DatePickerInput.tsx

```diff
@@ -7,10 +7,7 @@ import { Button } from '../button';
 import { Field } from '../field';
 import { parseDate, formatHumanReadableDate } from './utils';
 
-export type DateInputProps = Omit<
```

Comment:
We can remove all of the Omits here since `DatePickerProps` no longer relies on this 👍